Readme.md

```diff
@@ -30,7 +41,7 @@ Install
 ```bash
 # Replace my-gameserver with how you wish to name your game
 export NODE_ENV=development
-npm install mage --bootstrap --prefix my-gameserver
+npm install mage --create --prefix my-gameserver
```

Comment:
`prefix` is an NPM flag; while we could potentially enforce its use, I am not sure how that would end up (e.g. we might end up with a broken install).

Nevertheless, if you believe this would be a good addition, you should create a separate issue instead, since here all we are doing is fixing the documentation for what already exists.